monitoring/httpendpoint.go

```diff
@@ -68,6 +70,7 @@ func new() *MetricExporter {
 			panic(err)
 		}
 	}()
+	tmp.Metrics = metrics.NewPrometheus(prometheus.DefaultRegisterer)
```

Comment:
Bad variable naming `tmp`. Go best practices suggest to be more expressive. The variable holds a metric exporter object. Suggested variables names would be `metricExporter`, `exporter`, `me` (no, not `e` as that is reserved for errors).